### PR TITLE
Update: Read.cv

### PR DESCRIPTION
**Pull request by 5ouma**
### Description

Updated `Read.cv` details below.

- Capitalize the name from `read.cv` to `Read.cv`.
- Update the source URL to a more readable one.

### CODE REVIEW

File: icons/readdotcv.svg
Comment (adamrusted) [inline icons/readdotcv.svg:1]: Hey @5ouma - I'm seeing some [slight differences](https://github.com/simple-icons/simple-icons/assets/1830867/58510a97-b20d-46a2-ab1e-bbecf0e875b6) between your PR in black, and the source image in red. Might just be an optimisation issue, but would you mind retrying this?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/readdotcv.svg
Comment (5ouma) [reply icons/readdotcv.svg:1]: Thank you @adamrusted!
I retried modifying the image, and does it work?

readdotcv

Comment (adamrusted) [reply icons/readdotcv.svg:1]: Hey @5ouma - could you commit your changes to this branch, and I'll compare.

Comment (5ouma) [reply icons/readdotcv.svg:1]: I pushed! Could you check it, please @adamrusted?

Comment (adamrusted) [inline icons/readdotcv.svg:1]: Thanks for the update @5ouma - though unfortunately I'm still seeing some differences. Could you try building from source again?
image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/readdotcv.svg
Comment (5ouma) [reply icons/readdotcv.svg:1]: I think these differences are made when being compressed with SVGO.
How much should I compress the SVG file?

Comment (5ouma) [reply icons/readdotcv.svg:1]: This file isn't compressed and passed the `svglint`.
readdotcv

Comment (5ouma) [reply icons/readdotcv.svg:1]: Is there no need to compress?

Comment (adamrusted) [reply icons/readdotcv.svg:1]: If its passing the linter, it's usually good to go!

**Comment by 5ouma** [comment]
@adamrusted, it must be ready to merge this PR!

**Review by adamrusted — APPROVED** [review]

**Pull request merged**